frontend/dist/css/index.css

```diff
@@ -74,16 +98,59 @@ nav {
   opacity: 0;
 }
 
+#register {
+  height : 100%;
+  background-color: #bdbdbd;
+}
+
 @media screen and (max-width: 600px) {
+  .card-panel {
+    margin: 70px auto;
+  }
+
   .qc-member-name {
     display: none;
   }
 
-  .qc-brand-logo {
-    margin-left: 50px;
+  .qc-sign-in-github {
+    display: none;
   }
+
+  .qc-sign-in-linkedin {
+    display: none;
+  }
+}
+
+.social-button {
+  display: block;
+  margin-bottom: 15px;
+  height: 64px;
+  cursor: pointer;
 }
 
+.social-button > img {
+  padding: 16px;
+  border-right: 1px solid white;
+}
+
+.social-button-text {
+  position: absolute;
+  line-height: 64px;
+  font-size: 1.25em;
+  margin-left: 16px;
+}
+
+.github {
+  color: white;
+  background-color: rgb(23, 21, 22);
```

Comment:
Just out of curiosity: some background-color uses rgb and others use hex code. Is there a reason for this?

Reply:
FYI, this is a matter of preference because rgb and hex are just two different representation of the same thing, but usually it's better to stick with one representation :)